lib/features.js

```diff
@@ -88,6 +88,14 @@ const features = {
                  'capabilities or host volume mounts.',
     defaults: false,
     module: require('./features/dind')
+  },
+
+  websocketServer: {
```

Comment:
This is the name that will be referenced in:
```js
task.payload.features = {
  websocketServer: true
}
```
I think this is too generic... Let's make it `interactive` that's more intuitive.. .